static/app/views/replays/replays.tsx

```diff
@@ -107,10 +107,10 @@ function Replays({location}: Props) {
           />
           <Pagination
             pageLinks={pageLinks}
-            onCursor={(offset, path, searchQuery) => {
+            onCursor={(cursor, path, searchQuery) => {
               browserHistory.push({
                 pathname: path,
-                query: {...searchQuery, offset},
```

Comment:
This is the important rename. 
`{offset}` is the same as `{offset: offset}` which gets turned into `?offset=123` in the query string.
Should all be called cursor.